frontend/packages/console-dynamic-plugin-sdk/src/extensions/pvc.ts

```diff
@@ -0,0 +1,89 @@
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+import { CodeRef, EncodedCodeRef, UpdateExtensionProperties } from '../types';
+
+export namespace ExtensionProperties {
+  export type PVCCreateProp = {
+    /** label for the create prop */
+    label: string;
+    /** path for the create prop */
+    path: string;
+  };
+
+  export type PVCAlert = {
+    /** CodeRef for the corresponding Alert component. */
+    alert: EncodedCodeRef;
+  };
+  export type PVCAlertCodeRefs = {
+    alert: CodeRef<React.FC<{ pvc: any }>>;
```

Comment:
@vojtechszocs after rebasing upon all your fixes.
using `pvc: PersistentVolumeClaimKind` still does not work as the schema won't compile.